src/inmanta/data/__init__.py

```diff
@@ -3000,13 +3000,18 @@ class Parameter(BaseDocument):
     resource_id: m.ResourceIdStr = ""
     updated: Optional[datetime.datetime] = None
     metadata: Optional[JsonType] = None
-    expires: bool = True
+    expires: bool
 
     @classmethod
-    async def get_updated_before_active_env(cls, updated_before: datetime.datetime) -> list["Parameter"]:
+    async def get_updated_before_active_env(
+        cls, updated_before: datetime.datetime, only_expiring: bool = False
```

Comment:
The old implementation would only return parameters that are expiring. This new implementation adds the `only_expiring` argument with the default value set to False. This is a breaking change. Every location where this method is used that doesn't provide the `only_expiring` will not behave differently than. Let's change the default value to True to keep things backwards compatible.